docs/guide/en/migrating-yii2-bootstrap.md

```diff
@@ -1,21 +1,21 @@
-Migrating from yii-bootstrap3
+Migrating from Yii2 bootstrap
```

Comment:
I don't see it as being useful. Maybe remove this part of the guide?

Reply:
If it is good to remove it.